### ISSUE TITLE
Epwing Dictionaries not being read in scan

### ISSUE THREAD

**Issue by sinqbad** [goldendict/goldendict#1029]
Linux T440 4.14.48-2-MANJARO #1 SMP PREEMPT Fri Jun 8 20:41:40 UTC 2018 x86_64 GNU/Linux
KDE 5.12 

```
$ goldendict
QFileSystemWatcher::removePaths: list is empty
QFileSystemWatcher::removePaths: list is empty
```

I have dictd set up on Arch and the Spanish-English (dict-freedict-eng-spa) installed fine and goldendict can read it fine. I am trying to add in my EPWING dictionaries. In Edit->Dictionaries->Sources->Files I add the my dictionary folder and enable recursive I click rescan and then OK. There is no Japanese->Japanese dictionary or any other EPWINGS I have.

**Comment by Abs62** [comment]
Did your GD build with Epwing support?

**Comment by sinqbad** [comment]
I installed it through pacman.

**Comment by Abs62** [comment]
Look "Full-text search" page in Preferences dialog. Is "Epwing" line presented?

**Comment by sinqbad** [comment]
No. So that means the pacman version doesn't have it? Do I have to compile the project myself?

**Comment by sinqbad** [comment]
No, does that mean the pacman version doesn't have it? Do I have to compile it myself?

**Comment by Abs62** [comment]
Yes.

**Comment by sinqbad** [comment]
Do I have to explicitly put in EPWING support? I can't find instructions to do so.

**Comment by Abs62** [comment]
No. On the contrary, qmake key "no_epwing_support" is required to build GD without Epwing support. Read readme.md file to learn about all qmake keys and required libraries.
